src/test/java/team/serenity/model/managers/QuestionManagerTest.java

```diff
@@ -48,7 +48,7 @@ public void resetDataWithDuplicateQuestionThrowsDuplicateQuestionException() {
     }
 
     @Test
-    public void hasQuestionNullQuestion_throwsNullPointerException() {
+    public void hasQuestionNullQuestionThrowsNullPointerException() {
```

Comment:
I realised my test case naming convention is wrong for this file!! I will create a PR regarding this. It is suppose to have the underscores to differentiate (following the textbook naming convention)